### ISSUE TITLE
DisplayOnlyType throws error on form validation when null

### ISSUE THREAD

**Issue by grossmannmartin** [shopsys/shopsys#1867]
### What is happening
If I create a new `DisplayOnlyType` form field and the value of it is `null`

_for example in category_
```php
$builderSettingsGroup
    ->add('test', DisplayOnlyType::class, [
        'data' => null,
        'label' => t('Test'),
    ]);
```

and I disable Javascript validation – so the server side validation take place.

Then after submitting invalid form (for example required fields are not filled), the app ends up on error 500.

```
Twig\Error\RuntimeError

An exception has been thrown during the rendering of a template ("Notice: Array to string conversion") in "@ShopsysFramework/Admin/Form/displayOnlyFields.html.twig". search►
```

I notice the value passed into the template (packages/framework/src/Resources/views/Admin/Form/displayOnlyFields.html.twig) as a variable `data` is "somewhere", "somehow" converted to empty array.

### Expected result

I can use server side validation with null values in DisplayOnlyForm type.

**Comment by pk16011990** [comment]
Maybe it would be enough to have this field disabled by default.

**Comment by sslt** [comment]
related to https://github.com/shopsys/shopsys/issues/1943

**Comment by grossmannmartin** [comment]
closed by #2184
